src/screens/faq/index.tsx

```diff
@@ -0,0 +1,110 @@
+} from '@components';
+import Grid from '@material-ui/core/Grid';
+import {
+  createMuiTheme, MuiThemeProvider,
+} from '@material-ui/core/styles';
+import { useGetStyles } from './styles';
+
+const themeLocal = createMuiTheme({
+  breakpoints: {
+    keys: ['xs', 'sm', 'md', 'lg', 'xl'],
+    values: {
+      xs: 0, sm: 568, md: 760, lg: 1080, xl: 1200,
+    },
+  },
+});
+
+const Faq = () => {
+  const { t } = useTranslation('faq');
+  const { classes } = useGetStyles();
+
+  return (
+    <Layout>
+      <MaxWidthContainer>
+        <MaxWidthContent className={classes.root}>
+          <h1 className={classes.title}>
+            {t('FAQ')}
+          </h1>
+          <p className={classes.context}>
+            {t('context')}
+            <a
+              href="mailto:support@forbole.com"
+              className={classes.desktopOnly}
+            >
+              {t('email')}
+            </a>
+          </p>
+          <div className={classes.linkContainer}>
+            <a
+              href="mailto:support@forbole.com"
+              // style="Color:#1D86FF"
+              style={{
+                color: '#1D86FF',
+              }}
+            >
+              {t('email')}
+            </a>
+          </div>
+          <MuiThemeProvider theme={themeLocal}>
+            <Grid container spacing={2}>
+              <Grid item xs={12} lg={6}>
+                <h2 className={classes.subTitle}>
+                  {t('subTitle1')}
+                </h2>
+                <p className={classes.paragraph}>
+                  {t('p1')}
+                </p>
+              </Grid>
+              <Grid item xs={12} lg={6}>
+                <h2 className={classes.subTitle}>
+                  {t('subTitle2')}
+                </h2>
+                <p className={classes.paragraph}>
+                  {t('p2')}
+                </p>
+              </Grid>
+              <Grid item xs={12} lg={6}>
+                <h2 className={classes.subTitle}>
+                  {t('subTitle3')}
+                </h2>
+                <p className={classes.paragraph}>
+                  {t('p3')}
+                </p>
+              </Grid>
+              <Grid item xs={12} lg={6}>
+                <h2 className={classes.subTitle}>
+                  {t('subTitle4')}
+                </h2>
+                <p className={classes.paragraph}>
+                  {t('p4')}
+                </p>
+              </Grid>
+              <Grid item xs={12} lg={6}>
+                <h2 className={classes.subTitle}>
+                  {t('subTitle5')}
+                </h2>
+                <p className={classes.paragraph}>
+                  {t('p5')}
+                </p>
+              </Grid>
+              <Grid item xs={12} lg={6}>
+                <h2 className={classes.subTitle}>
+                  {t('subTitle6')}
+                </h2>
+                <p className={classes.paragraph}>
+                  {t('p6')}
+                </p>
```

Comment:
This will get messy really fast if we decide to add/remove questions/answers. One way we can create neater code is by creating a `utils.ts` in `src/screens/faq/utils.ts` and inside the file we will have something like this
```
const faqs = [
  {
    question: 'question1', // the i18n key
    answer: 'answer1', // the i18n key
  },
  {
    question: 'question2',
    answer: 'answer2'
  }
]
```
This will allow you or any other users editing this this to quickly insert/delete/edit the config without having to worry too much about the jsx syntax
and in the `index.tsx` something like this (reference only i copy and pasted from some random file)
```
{faqs.map((x, i) => {
            return (
              <ListItem
                key={i}
              >
                {x}
                {i !== linkComponents.length - 1 && (
                  <span className={classnames('list-decorator')}>{linkDecorator}</span>
                )}
              </ListItem>
            );
          })}
```

Reply:
done